examples/go-client/token.go

```diff
@@ -1,53 +0,0 @@
-package main
```

Comment:
Why'd you remove these examples? Won't this break the documentation that depends on these examples or are thees inlined into the documentation now?

Reply:
These aren't referenced in the documentation, they are inlined. I think it was a mistake by me to have these examples here in the first place. Having these examples in the go-client example is redundant and distracts from the purpose of the example, which is to get the go-client up and running with the authentication methods. It also demands more upkeep. The documentation should be the go to resource for actual example and code help, whereas the go-client example should be a starting point for any api usage.